src/structures/Integration.js

```diff
@@ -125,17 +125,12 @@ class Integration extends Base {
    * Sync this integration
    * @returns {Promise<Integration>}
    */
-  sync() {
+  async sync() {
     this.syncing = true;
-    return this.client.api
-      .guilds(this.guild.id)
-      .integrations(this.id)
-      .post()
-      .then(() => {
-        this.syncing = false;
-        this.syncedAt = Date.now();
-        return this;
-      });
+    await this.client.api.guilds(this.guild.id).integrations(this.id).post();
+    this.syncing = false;
```

Comment:
This might be better in a finally, if something goes wrong. ~~Not that anyone uses this method.~~
I also think this endpoint is no longer available to bots. 🤔 (For another PR though)